backend/cmd/server.go

```diff
@@ -17,6 +18,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	if conf.ListPlugins {
+		if err := plugins.ListPlugins(conf.StaticDir, conf.PluginsDir); err != nil {
+			logger.Log(logger.LevelError, nil, err, "error listing plugins")
```

Comment:
Small NIT. Since this is already an error, adding error keywords adds redundancy. 

```suggestion
			logger.Log(logger.LevelError, nil, err, "listing plugins")
```

Reply:
Great catch! Removed these redundancies